src/silx/gui/colors.py

```diff
@@ -1059,6 +1076,10 @@ def preferredColormaps() -> tuple[str, ...]:
     if _PREFERRED_COLORMAPS is None:
         # Initialize preferred colormaps
         setPreferredColormaps(_DEFAULT_PREFERRED_COLORMAPS)
+
+    # setPreferredColormaps initialize _PREFERRED_COLORMAPS
+    assert _PREFERRED_COLORMAPS is not None
```

Comment:
There is already a `ValueError` in `setPreferredColormaps`, so I guess this is for type linter, otherwise it is not needed

```suggestion
    if _PREFERRED_COLORMAPS is None:
        raise RuntimeError("_PREFERRED_COLORMAPS should not be None")
```

Reply:
Yes, that's for the type checking.

Actually there is no better solution, except reworking the code.